src/test/java/maksym/fedorenko/bookstore/repository/BookRepositoryTest.java

```diff
@@ -5,67 +5,30 @@
 
 import com.querydsl.core.types.Predicate;
 import java.math.BigDecimal;
-import java.sql.Connection;
 import java.util.List;
 import java.util.Optional;
-import javax.sql.DataSource;
-import lombok.SneakyThrows;
 import maksym.fedorenko.bookstore.model.Book;
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
 import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
-import org.springframework.core.io.ClassPathResource;
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.PageRequest;
-import org.springframework.jdbc.datasource.init.ScriptUtils;
+import org.springframework.test.context.jdbc.Sql;
 
 @DataJpaTest
 @AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
```

Comment:
do you really need this? `NONE` means use autoconfigured datasource, which is H2 here

therefore I guess you can skip this annotation - pls verify

**Applicable to all such tests**